viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+
+
+
+from webbrowser import get
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie = None
+    if title == None or genre == None or rating == None:
+        movie == None
+    else:
+        movie = {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if title in movie["title"]:
+            user_data["watchlist"].remove(movie)

```

Comment:
One danger of modifying the `user_data` while iterating is that it could cause an error in our loop. Try to avoid modifying a list while you're iterating through it!